lib/dry/validation/hint_compiler.rb

```diff
@@ -61,6 +61,11 @@ def visit_key(node)
         name
       end
 
+      def visit_attr(node)
+        name, _ = node
```

Comment:
I would leave it as is just for sake of readability. It's not obvious what is hidden under the `node.first` at the first look.